maxatac/utilities/parser.py

```diff
@@ -240,7 +240,7 @@ def get_parser():
     predict_parser.add_argument("-bl", "--blacklist",
                                 dest="blacklist",
                                 type=str,
-                                help="The blacklisted regions to exclude in BED format"
+                                help="The blacklisted regions to exclude in bigWig format"
```

Comment:
This is incorrect. If you look in predict.py these should be in BED format

Reply:
Ah, this makes sense -- it looks like maybe the documentation is incorrect for predict, then? It currently reads as follows:
<img width="673" alt="predict_documentation_screenshot" src="https://github.com/user-attachments/assets/a2f9c976-b39d-4c02-8faa-05f1fecbd0e3">